packages/core/src/forms/studio/formula.helpers.ts

```diff
@@ -52,20 +52,16 @@ export const mapStudioFieldsWithFormula = (
   }));
 };
 
-export const getAttrsValue = (object: any) => {
-  if (isEmpty(object)) {
+export const getAttrsValue = (object: Object, fieldType: string) => {
+  if (
+    isEmpty(object) ||
+    !object.hasOwnProperty(fieldType) ||
+    typeof object[fieldType] !== 'string'
+  ) {
     return {};
   }
 
-  let result = {};
-
-  Object.entries(object)
-    .filter(([key]) => key.toLowerCase().includes('attrs'))
-    .forEach(([_, value]: [string, string]) => {
-      result = {...result, ...JSON.parse(value)};
-    });
-
-  return result;
+  return JSON.parse(object[fieldType]);
```

Comment:
That was not the objective of this ticket. fieldType is not required on CustomFieldForm so it should not be required here too. 
Wanted behaviour : 
* if object is empty : return empty object
* if fieldType is not null and object[fieldType] is not null then return object[fieldType]
* else keep current behaviour to return values of attrs fields